lib/representer.ex

```diff
@@ -6,7 +6,7 @@ defmodule Representer do
   def process(file, code_output, mapping_output) do
     {represented_ast, mapping} = represent(file)
 
-    File.write!(code_output, Macro.to_string(represented_ast))
+    File.write!(code_output, Macro.to_string(represented_ast) <> "\n")
```

Comment:
My IDE is configured to auto-add trailing new lines to all files that I save. It's a good practice so that the files can be nicely read in a terminal without the prompt being printed on the same line as the last line of the file.

To be able to keep my settings and work in the IDE, I would need to add this tiny innocent new line to the produced representation 😇